v1/references.yaml

```diff
@@ -155,7 +155,8 @@ definitions:
     description: A single reference list
     properties:
       id:
-        type: string
+        type: ['string', 'null']
+        nullable: true
```

Comment:
Is `nullable` needed if `null` is one of the possible types/values?

Reply:
nullable is not needed yet. I just kept it so we don't forget to add it when we move to Swagger 3.0 or higher.